lib/support/resultsStorage/index.js

```diff
@@ -26,7 +26,11 @@ module.exports = function(input, timestamp, outputFolder, resultBaseURL) {
     storageBasePath = path.resolve(outputFolder);
   } else {
     resultsSubFolders.push(getDomainOrFileName(input), timestamp.format('YYYY-MM-DD-HH-mm-ss'));
-    storageBasePath = path.resolve('sitespeed-result', ...resultsSubFolders);
+    if (resultBaseDir) {
+      storageBasePath = path.resolve(resultBaseDir,'sitespeed-result', ...resultsSubFolders);
```

Comment:
According to the cli description, the sitespeed-result should not be added to this path.